File: py/tutorials/codewars/string_list/split_strings.js
function solution(str){
  var l = str.length; 
  var arr = [];
  
  if (l%2==1){
    str = str+'_';
  }
  
  for(var i=0;i<l;i=i+2){  
    var val = str.slice(i,i+2);
    arr.push(val);}
  
  return arr;
  
}

File: py/tutorials/codewars/string_list/split_strings.py
def solution(s):
    out = None
    if len(s)%2==1:
        s = s+'_'
        out = [s[k:k+2] for k in range(0,len(s),2)]
    else:
        out = [s[k:k+2] for k in range(0,len(s),2)]
    return out